content/millennium/r4/financial/general/charge-item.md

```diff
@@ -157,6 +157,53 @@ List an individual ChargeItem by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Operation: charge-item-credit
+
+<%= beta_tag(action: true) %>
+
+Creates an offsetting ChargeItem for an existing debit ChargeItem.
+
+    POST/ChargeItem/:id/charge-item-credit
+    
+### Authorization Types
+
+<%= authorization_types(provider: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/fhir+json', 'Content-Type': 'application/fhir+json'} %>
+
```

Comment:
I think we will probably need to provide notes and description for each of the parameter Body Fields, just like we do for the fields for other resources.
@alexbeckner1

Reply:
Done i have made the changes like this https://github.com/cerner/fhir.cerner.com/pull/726/commits/71fbf617bf8defa672134577e046925aae99cedd#diff-3922401d7ff96de834a6f5f9a2f3f547eb8c325edfd903beb40c7773268f67e0R1-R111 is this fine @alexbeckner1